hogben/utils.py

```diff
@@ -138,7 +138,7 @@ def __corner(self, results):
         axes[self.ndim-1, self.ndim-1].set_xlabel(self.params[-1].name)
         return fig
 
-def fisher(qs, xi, counts, models, step=0.005):
+def fisher(qs, xi, counts, models, step=0.005, importance = None):
```

Comment:
Can you add typing into the definitions (and remove them from the docstrings) as you touch the code